src/argilla/client/client.py

```diff
@@ -104,11 +104,28 @@ def __init__(
                 the headers of argilla client requests, like additional security restrictions. Default: `None`.
 
         """
-        api_url = api_url or os.getenv("ARGILLA_API_URL", "http://localhost:6900")
+        from argilla.client.login import ArgillaCredentials
+
+        api_url = api_url or os.getenv("ARGILLA_API_URL")
+        api_key = api_key or os.getenv("ARGILLA_API_KEY")
+        workspace = workspace or os.getenv("ARGILLA_WORKSPACE")
+        extra_headers = extra_headers or {}
+
+        if api_url is None and api_key is None and workspace is None:
+            try:
+                credentials = ArgillaCredentials.load()
+                api_url = credentials.api_url
+                api_key = credentials.api_key
+                workspace = credentials.workspace
+                extra_headers = credentials.extra_headers
+            except FileNotFoundError:
+                pass
+
+        api_url = api_url or "http://localhost:6900"
```

Comment:
Should we defined the default host as `DEFAULT_API_URL`? I assume we won't change the port at any point, but since this is a default value, aren't we more safe if we just add it under `argilla._constants`?

Reply:
sure